docs/dev/new_check_howto.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Create a New Integration
+title: Create an Agent integration
```

Comment:
I'm going to come back and add some further reading pages once we're done with the marketplace docs